Dockerfile

```
@@ -1,5 +1,10 @@
 FROM node:6
-MAINTAINER Adam M. Brown <adam.brown@hq.dhs.gov>
+
+# For a list of pre-defined annotation keys and value types see:
+# https://github.com/opencontainers/image-spec/blob/master/annotations.md
+# Note: Additional labels are added by the build workflow.
+LABEL org.opencontainers.image.authors="Adam M. Brown, nicholas.mcdonnell@cisa.dhs.gov"
```

I thought we were leaning toward listing the Fusion Dev Group as an author for Docker containers, unless there was a compelling reason not to.  That's what I've been doing, e.g., [here](https://github.com/cisagov/scanner/blob/develop/Dockerfile#L9).

```suggestion
LABEL org.opencontainers.image.authors="Adam M. Brown, vm-fusion-dev-group@trio.dhs.gov"
```

---

Yes, sorry that change was from before we made that decision and I forgot to update it based on the latest best practices.